### ISSUE TITLE
empty repo

### ISSUE THREAD

**Issue by SheepTester** [anematode/sigfigjam#1]
idk how to run help pls my homework is due tomorrow and i need this now!!!

**Comment by ghost** [comment]
yes pls fiz fast i ned fpr project need quik or else will fial and wont go to gud colege and have god life pls i need this pls pls
